File: WYNKWeb/src/app/Models/ViewModels/IOProcedureTemplateViewModel.ts
export class IOProcedureTemplateViewModel {

  IOProcedureTemp: IOProcedureTemp = new IOProcedureTemp();
  IOProcedureTempDetails: Array<IOProcedureTemp> = [];
}


export class IOProcedureTemp {
  ID: number;
  icdSpeciality: string;
  SurgeryDescription: string;
  OTNotesDescription: string;
  Status: boolean;
  icdSpecialityDesc: string;
  UserInputType: string;
  InputValue: string;

  changed: boolean;
  Added: boolean;
}

File: WYNK.Data/Model/ViewModel/IOProcedureTemplateViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WYNK.Data.Model.ViewModel
{
    public class IOProcedureTemplateViewModel
    {
     public IOProcedureTemp IOProcedureTemp { get; set; }
      public ICollection<IOProcedureTemp> IOProcedureTempDetails { get; set; }
    }

    public class IOProcedureTemp {
        public string icdSpeciality { get; set; }
        //public string Eye { get; set; }
        public string SurgeryDescription { get; set; }
        public string OTNotesDescription { get; set; }
        public string UserInputType { get; set; }
        public string InputValue { get; set; }
        public Boolean Status { get; set; }

        public int? ID { get; set; } 
        public Boolean changed { get; set; } 
        public Boolean Added { get; set; } 
    }




}

